src/Tribe/RSVP.php

```diff
@@ -1046,6 +1046,8 @@ public function front_end_tickets_form( $content ) {
 			}
 		}
 
+		$must_login = ! is_user_logged_in() && $this->login_required();
```

Comment:
@bordoni I requested you for this - this was being defined after it was being passed to the filter on line 1071